vsdownload.py

```diff
@@ -88,12 +97,34 @@ def setPackageSelectionMSVC15(args, packages, userversion, sdk, toolversion, def
         args.package.extend(defaultPackages)
 
 def setPackageSelection(args, packages):
+    if not args.architecture:
+        if args.host_arch is not None:
+            args.architecture = [ args.host_arch ]
```

Comment:
Hmm. I'm a little divided on this; changing this will affect existing users that run `vsdownload.py` expecting to get all cross compilers installed. I can see the case that many users only expect the host arch though.